### DISCUSSION TITLE
Office_S_Tips/system/2020/02/20/Win10-klucz_i_kopia_sterownikow

### DISCUSSION THREAD

**Discussion by giscus** [andrzejQ/Office_S_Tips#27] [Komentarze]
# Office_S_Tips/system/2020/02/20/Win10-klucz_i_kopia_sterownikow

Klucz licencyjny Windows 10 jest na ogół zachowany w BIOS.

https://andrzejq.github.io/Office_S_Tips/system/2020/02/20/Win10-klucz_i_kopia_sterownikow.html

**Comment by andrzejQ** [comment]
Nowa wersja komentarzy z użyciem `giscus -> GitHub Discussions`.
